dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/ApplicationProcessorTest.java

```diff
@@ -72,85 +85,222 @@ public static void setUpTest() throws Exception {
      */
     @Before
     public void setUp() throws Exception {
-        ServerConfiguration config = server.getServerConfiguration();
-        config.getApplications().stream().forEach(app -> OpenAPITestUtil.removeApplication(server, app.getName()));
-        server.updateServerConfiguration(config);
+        // Remove all the deployed apps from server.xml
+        OpenAPITestUtil.removeAllApplication(server);
+
+        // Change server ports to the default ones
+        OpenAPITestUtil.changeServerPorts(server, server.getHttpDefaultPort(), server.getHttpDefaultSecurePort());
     }
 
     @AfterClass
     public static void tearDown() throws Exception {
-        server.stopServer("CWWKO1650E");
+        server.stopServer("CWWKO1650E", "CWWKO1651W");
```

Comment:
Are the tests using invalid apps to ignore validation errors/warning? It may be fine since this test's focus is not on validation. But I recall yesterday's discussion (involving Anna) about running a valid app and making sure no errors/warning thrown by validator. Is this covering that case as well or is there a different test for that?

Reply:
Anna fixed the validation issues in the Airlines app, but in this test I'm using other apps which are throwing validation errors/warnings.
I'll change the app in the ProxySupportTest to Airlines and take out the error code from there.

Reply:
Ok. sounds good. Thanks.